python/servo/try_parser.py

```diff
@@ -89,12 +89,22 @@ def handle_preset(s: str) -> Optional[JobConfig]:
 
     if s == "linux":
         return JobConfig("Linux", Workflow.LINUX, unit_tests=True)
-    elif s in ["perf", "linux-perf", "bencher"]:
+    elif s in ["linux-perf"]:
         return JobConfig("Linux perf", Workflow.LINUX, bencher=True)
+    elif s == "linux-prod-perf":
+        return JobConfig("Linux prod perf", Workflow.LINUX, bencher=True, profile="production")
```

Comment:
"prod perf" is very cryptic. I think instead of adding more custom names, we should simply allow the caller to add more clarification. For instance: "linux +production +bencher" In addition, the job names should be printed with a similar kind of clarity. The text of the job in the GitHub interface should look like this "Linux (Bencher, Production)" so it's clear that these are modifiers to the "Linux" job.

Reply:
Initial try_parser had support for configuring job options, but it was complicated (and not really useful). I suggest we use this syntax: `preset+modifier1+modifier2` to keep modifiers local on job (preset).

Reply:
@mrobinson looks like merge is blocked by this request. Could you review it?